CODE/Logicytics.py

```
@@ -67,35 +74,31 @@
         return output, "info"
 
 
-@log.function
-def get_flags() -> tuple[str, str]:
+def get_flags():
```

Ruff (0.8.0)

Missing return type annotation for public function `get_flags`

Add return type annotation: `None`

(ANN201)